src/definitions/modules/calendar.js

```diff
@@ -884,10 +890,10 @@
                             return false;
                         }
 
-                        var endDate = module.get.endDate();
+                        let endDate = module.get.endDate();
                         if (!!endDate && !!date && date > endDate) {
                             // selected date is greater than end date in range, so clear end date
-                            module.set.endDate(undefined);
+                            module.set.endDate();
```

Comment:
please review/test [fix unicorn/no-useless-undefined todo](https://github.com/fomantic/Fomantic-UI/pull/2584/commits/1db3efe49437333b0a5a6d5e9e53dbacaec491fc) commit (this line and another 1 line in this file)

it should be fine but I am not sure if the `module.set.endDate` function does not need `undefined` strictly like for counting the passed arguments count

Reply:
fixed in #2591